src/provider/postMessage.js

```diff
@@ -172,12 +170,6 @@ class PostMessage extends EventEmitter {
       return;
     }
 
```

Comment:
Reverted the Logging attachment - PostMessage has subscriptions, so no need to double-up.